bin/create_folders.py

```diff
@@ -39,24 +40,24 @@ def build_subfolder(
 def update_permissions(
     syn: synapseclient.Synapse,
     subfolder: Union[str, synapseclient.Entity],
-    project_folder_id: str,
+    project_id: str,
```

Comment:
Out of scope, but I renamed the `project_folder_id` arg to `project_id` because I felt the previous name was confusing